packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -611,6 +623,14 @@ class SkiaGoldClient {
     ));
     return authFile.existsSync();
   }
+
+  /// Returns the contents of the authorization configuration.
+  Future<String> _printClientAuthConfig() async {
+    return workDirectory.childFile(fs.path.join(
+      'temp',
+      'auth_opt.json',
```

Comment:
I'm assuming this doesnt' contain secrets? :)

Reply:
Nope. This is sample contents: 
```json
{
"Luci":false,
"ServiceAccount":"/dev/goldTesting/flutter-infra-56c2.json",
"GSUtil":false
}
```

I want to see what it is indicating for gsutil in particular.
But I think I will also add output for the `client_email` of the service account, which is what seems to be getting mixed up.